backends/ubpf/p4include/ubpf_model.p4

```diff
@@ -99,6 +99,10 @@ extern Register<T, S> {
  */
 extern bit<48> ubpf_time_get_ns();
 
+/*
+ * Truncate packet
+ */
+extern void truncate(in bit<32> len);
```

Comment:
It would be good to have a sentence or three describing at least the units of the parameter `len` (bits?  bytes?), whether it starts from the first byte of an Ethernet header, if the packet is Ethernet, or some other location, and whether the intent is to support any integer number value of `len`, or only some subset of those values are supported, e.g. only multiples of 16 bytes.